src/main/java/dk/kb/netarchivesuite/solrwayback/export/GenerateCSV.java

```diff
@@ -72,8 +75,15 @@ public  static void generateLine(StringBuffer buffer,SolrDocument doc, String[]
                 if (field_value instanceof List) { //if multivalued
                     field_value = String.join(MULTIVALUE_SEPARATOR, (List<String>) field_value);
                 }
-                String escaped = escapeQuotes(field_value.toString());
-                result.append(escaped);
+                if (field_value instanceof String) {
```

Comment:
Why not also "else if" here also for clarity? (and very minor optimization)

Reply:
Fair point. I have made an unbroken `if-else` chain and adjusted the code so that it follows the same style in each of the code snippets